graphics/cairo-devel/Portfile

```diff
@@ -101,6 +101,14 @@ platform macosx {
     # Don't allow Quartz support to be disabled. Keep the variant for awhile in
     # case any dependents are using the active_variants portgroup to check for it.
     variant_set             quartz
+
+    # Apply changes from https://gitlab.freedesktop.org/cairo/cairo/-/merge_requests/52
+    # on macOS Big Sur and newer. Fixes:
+    # https://gitlab.freedesktop.org/cairo/cairo/-/issues/420
+    # https://trac.macports.org/ticket/61586
+    if {${os.major} > 19} {
```

Comment:
Why apply the patch conditionally? Upstream has merged the code unconditionally for all platforms; we should use it for all platforms too.

Reply:
The main reason is that I cannot test anything else than Big Sur and the old version didn't cause problems on older macOS versions. Feel free to apply it unconditionally.